components/admin_console/group_settings/groups_list/groups_list.tsx

```diff
@@ -359,7 +389,7 @@ export default class GroupsList extends React.PureComponent {
     }
 
     resetFiltersAndSearch = () => {
-        const newState = {
+        const newState: any = {
```

Comment:
You can use `Partial<State>` here

Reply:
Used Partial, but have to case explicitly to any in setState